cmake/FindSSE.cmake

```diff
@@ -0,0 +1,148 @@
+# Check if SSE/AVX instructions are available on the machine where
```

Comment:
This seems not a good criterion, unless it is merely for use as a default. I suggest to add to the comment right here how to override it when calling CMake.